src/sentry/integrations/jira/integration.py

```diff
@@ -147,6 +149,19 @@ class JiraIntegration(IssueSyncIntegration):
     resolution_strategy_key = "resolution_strategy"
     comment_key = "sync_comments"
 
+    def raise_error(self, exc: Exception, identity: Identity | None = None) -> NoReturn:
+        if isinstance(exc, ApiInvalidRequestError):
+            if exc.json:
+                error_fields = self.error_fields_from_json(exc.json)
+                if error_fields is not None:
+                    raise IntegrationFormError(error_fields).with_traceback(sys.exc_info()[2])
+
+            # Even if Jira's error response isn't standard, we should still raise an error for the frontend
+            # TODO(iamrajjoshi): Rename this error to something better because we don't use it for what it's named for
+            raise IntegrationInstallationConfigurationError from exc
```

Comment:
would `IntegrationFormError` work here? It looks like the Sentry error is from people not filling out the create issue  form correctly (?)

Reply:
The format being -
```
{
"errorMessages": ["Issue can be assigned only active or future sprints."],
"errors": {},
},
```
?

It looks like the 'regex' handler from `self.error_fields_from_json` tries to do this but is v. limited in the fields it detects. Could we update `self.error_fields_from_json` instead to spit out some generic format (e.g fields : [array of error messages here])